frontend/css/layout.css

```diff
@@ -64,12 +100,67 @@ li a {
     padding-top: var(--spacing-m);
 }
 
+.search-icons {
+    flex-direction: column;
+    align-items: center;
+}
+
 .ideas {
     text-align: center;
 }
 
 /* Categories section */
```

Comment:
Rather than hints in comments to remind you what this styling is for (`/* Categories section */`) could the css classes be more explicit so you can shuffle code around without losing track of what it's there for? e.g. instead of `.sidebar` `.categories .sidebar`?

Reply:
Good point, some of these sections may be possible to extract into separate files? And/or apply a BEM approach to them, similar to how we did it for the menu and menu items previously?